src/components/badges/BadgeBlock.tsx

```diff
@@ -17,7 +17,7 @@ function Badge({
   onMinted,
   onMintFailed,
 }: {
-  proof: BaseProof
+  proof: Proof
```

Comment:
Lets extracts this props/args into `interface` to re-use it across default export and suspended component

Reply:
Let's extract it into `models`, look here: https://github.com/BigWhaleLabs/seal-cred-frontend/pull/455/files#r954740241

Reply:
not sure